api/answer/ConceptMap.java

```diff
@@ -39,6 +41,13 @@ public interface ConceptMap {
     @CheckReturnValue
     Concept get(String variable);
 
+    @CheckReturnValue
+    default JsonObject toJSON() {
+        JsonObject object = Json.object();
+        map().forEach((resVar, resConcept) -> object.add(resVar, resConcept.toJSON()));
+        return object;
+    }
```

Comment:
All implementations of `toJSON()` are done in the `api` package. Otherwise the implementation would be duplicated between e.g. `ThingImpl` and `ThingImpl.Remote`, since `...Impl.Remote` don't extend their enclosing counterparts.